packages/admin/src/components/bindingFacade/richText/baseEditor/BlockElement.tsx

```diff
@@ -18,11 +20,14 @@ export const BlockElement = memo(function BlockElement({
 	const editor = useSlateStatic()
 	const dataAttributes = ContemberEditor.getElementDataAttributes(element)
 	const El = domElement
+	const formatter = useMessageFormatter(editorDictionary)
+
 	return (
 		<El {...dataAttributes} {...attributes}>
 			{withBoundaries && (
 				<EditorBlockBoundary
 					blockEdge="before"
+					newParagraphText={formatter('editorBlock.editorBlockBoundary.label.newParagraph')}
```

Comment:
Maybe make it shorter?
```suggestion
					newParagraphText={formatter('editorBlock.editorBlockBoundary.newParagraph')}
```